Makefile

```diff
@@ -47,10 +47,11 @@ dist/docker/mesos-calico.tar: docker_image
 
 jenkins: calico_mesos
 	docker build -t calico/mesos-calico .
-
-jenkins-vagrant: calico_mesos docker_image.tar
-	vagrant up
-
+	export MESOS_CALICO_TAR=true
+	docker build -t calico/mesos-calico .
+	mkdir -p dist/docker
+	docker save -o dist/docker/mesos-calico.tar calico/mesos-calico
+	vagrant up --provider=vsphere
```

Comment:
This is the only command that makes use of `MESOS_CALICO_TAR`, right? instead of setting the variable, just prefix this command:
```
MESOS_CALICO_TAR=true vagrant up --provider=vsphere
```